packages/console/src/pages/ApiResourceDetails/index.tsx

```diff
@@ -70,22 +71,33 @@ function ApiResourceDetails() {
     }
   };
 
-  return (
-    <div
-      className={classNames(detailsStyles.container, isOnPermissionPage && styles.permissionPage)}
-    >
-      <TextLink to="/api-resources" icon={<Back />} className={styles.backLink}>
-        {t('api_resource_details.back_to_api_resources')}
-      </TextLink>
-      {isLoading && <DetailsSkeleton />}
-      {error && (
+  if (isLoading) {
+    return (
+      <DetailsPageWrapper backLink={backLink} backLinkTitle={backLinkTitle}>
+        <DetailsSkeleton />
+      </DetailsPageWrapper>
+    );
+  }
+
+  if (error) {
+    return (
+      <DetailsPageWrapper backLink={backLink} backLinkTitle={backLinkTitle}>
         <RequestDataError
           error={error}
           onRetry={() => {
             void mutate();
           }}
         />
-      )}
+      </DetailsPageWrapper>
+    );
+  }
```

Comment:
Is it possible to put `isLoading` and `hasError` logic in the `DetailsPage` component?